src/main/java/com/google/sps/servlets/getDailyQuiz.java

```diff
@@ -0,0 +1,61 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import java.lang.reflect.Type;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.QueryResultList;
+import java.text.DateFormat;
+import java.util.Date;
+import com.google.appengine.api.datastore.Key;
+
+@WebServlet("/getQuizStatus")
+public class getDailyQuiz extends HttpServlet {
+
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        
+        Query query = new Query("Quiz");
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        PreparedQuery pq = datastore.prepare(query);
+
+        Object time = null;
+        for(Entity entity : pq.asIterable()) {
+            time = entity.getProperty("timestamp");
+        }
+
+        FetchOptions fetchOptions = FetchOptions.Builder.withLimit(1);        
+        List<Entity> quizEntities = pq.asQueryResultList(fetchOptions);
```

Comment:
do u have to return it as a list if there is only 1 object you're fetching at a time?

Reply:
I am not sure how to save it without errors if it is not in a list, do you know how this might be done? But also, I was thinking that sending a list to js makes everything easier to use/manipulate so keeping it a list might be good

Reply:
One way to do it would be pq.asQueryResultList(fetchOptions)[1]; thats what i did in my code. if u think keeping a list is easier, thats fine too.